client/e2e/setup/global.ts

```diff
@@ -0,0 +1,196 @@
+          ON CONFLICT (user_guid)
+          DO UPDATE SET
+            app_role_id = EXCLUDED.app_role_id;
+        `;
+        await pool.query(upsert, [
+          process.env.E2E_CAS_USER_GUID,
+          user,
+          `${user}@test.com`,
+          role,
+        ]);
+        break;
+    }
+    // 🛸 Navigate to home page
+    const homePage = new HomePOM(page);
+    await homePage.route();
+    // 🔑 Login
+    // eslint-disable-next-line no-console
+    console.log(`🔑 Logging in user ${user}`);
+    await homePage.login(user, password, role);
+    await homePage.userIsLoggedIn();
+    // 💾 Capture the storage state (e.g., auth session cookies) of the current page and saves it to a file specified
+    // This storeageState can then be used for e2e tests requiring authentication
+    await page.context().storageState({ path: storageState });
+    // eslint-disable-next-line no-console
+    console.log(
+      `🤸 Successful authentication setup for ${user} captured in storageState ${storageState} 🤸`,
+    );
+    await browser.close();
+  } catch (error) {
+    // Handle any errors that occurred during the authentication process
+    await browser.close();
+    // eslint-disable-next-line no-console
+    console.error(`Authentication failed for ${user}:`, error);
+    throw error;
+  }
+};
+
+export default async function globalSetup() {
+  // 🌍 Perform global setup tasks here...
+
+  // 🥞 Set DB for e2e tests pre-reqs
+  // eslint-disable-next-line no-console
+  console.log("🥞 Db setup for e2e tests environment...");
+  try {
+    let query: {
+      text: string;
+      values: (string | number)[];
+    } = {
+      text: `
+    INSERT INTO erc.operator (id, status, legal_name, trade_name, cra_business_number, bc_corporate_registry_number, business_structure, mailing_address_id, physical_address_id)
+    VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)
+    ON CONFLICT (id)
+    DO UPDATE SET status = EXCLUDED.status;
+  `,
+      values: [
+        2,
+        "Approved",
+        "Existing Operator 2 Legal Name",
+        "Existing Operator 2 Trade Name",
+        "987654321",
+        "def1234567",
+        "BC Corporation",
+        4,
+        3,
+      ],
+    };
+    // eslint-disable-next-line no-console
+    console.error("✔️ Upserted Operation id 2 with status Approved.");
+    query = {
+      text: `
+    INSERT INTO erc.user_operator (user_id, role, status, operator_id)
+    VALUES ($1, 'admin', 'Approved',2)
+    ON CONFLICT (user_id)
+    DO UPDATE SET role = EXCLUDED.role, status = EXCLUDED.status;
+  `,
+      values: [process.env.E2E_INDUSTRY_USER_ADMIN_GUID as string],
+    };
+    // eslint-disable-next-line no-console
+    console.error(
+      "✔️ Upserted bc-cas-dev as user operator for operator_id 2 with role admin and status Approved.",
+    );
+    query = {
+      text: "DELETE FROM erc.user WHERE user_guid = $1",
+      values: [process.env.NEW_USER_GUID as string],
+    };
+    // Execute the deletion query
```

Comment:
Where does cas-three get added? I only see a manual add for an operator and user_operator.

I think we could also use some documentation about to set up the db in this file. Once we start writing workflow tests, people are going to need to clear and add stuff to test whatever workflow they're working on

Reply:
`client/e2e/workflows/bceidbusiness/new_user.spec.ts`